api/src/main/resources/hibernate.cfg.xml

```diff
@@ -68,7 +68,7 @@
 		<mapping resource="org/openmrs/api/db/hibernate/PatientIdentifierType.hbm.xml" />
                 <mapping resource="org/openmrs/api/db/hibernate/PatientProgram.hbm.xml" />
                 <mapping resource="org/openmrs/api/db/hibernate/PatientProgramAttribute.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/Relationship.hbm.xml" />
```

Comment:
@joelss-tech , 
i would gues this also implies removing the actual hibernate file , not just the mapping